src/app/compiler.js

```diff
@@ -210,10 +210,7 @@ function Compiler (editor, renderer, queryParams, handleGithubCall, outputField,
           continue;
         }
         if (editor.hasFile(m)) {
-          files[m] = window.localStorage[utils.fileKey(m)];
-          reloop = true;
-        } else if (m.startsWith('./') && editor.hasFile(m.slice(2))) {
```

Comment:
I'm lumping this into this Pull Request because I realized this code is now dead thanks to c2047b3b9482411446303bb8882d9f4d19421a29, which strips off leading `./`.